v1/ast/annotations.go

```diff
@@ -18,12 +18,32 @@ import (
 
 const (
 	annotationScopePackage     = "package"
-	annotationScopeImport      = "import"
 	annotationScopeRule        = "rule"
 	annotationScopeDocument    = "document"
 	annotationScopeSubpackages = "subpackages"
 )
 
+var (
+	scopeTerm            = StringTerm("scope")
+	titleTerm            = StringTerm("title")
+	entrypointTerm       = StringTerm("entrypoint")
+	descriptionTerm      = StringTerm("description")
+	organizationsTerm    = StringTerm("organizations")
+	authorsTerm          = StringTerm("authors")
+	relatedResourcesTerm = StringTerm("related_resources")
+	schemasTerm          = StringTerm("schemas")
+	customTerm           = StringTerm("custom")
+	refTerm              = StringTerm("ref")
+	nameTerm             = StringTerm("name")
+	emailTerm            = StringTerm("email")
+	schemaTerm           = StringTerm("schema")
+	definitionTerm       = StringTerm("definition")
+	documentTerm         = StringTerm(annotationScopeDocument)
+	packageTerm          = StringTerm(annotationScopePackage)
+	ruleTerm             = StringTerm(annotationScopeRule)
+	subpackagesTerm      = StringTerm(annotationScopeSubpackages)
```

Comment:
So technically, we're shifting work done multiple times (allocating and creating these terms) _on demand_ to having that work be done _once_ and sort-kinda at startup. Which would be negligible for `opa run -s`, but could be noticeable for `opa eval`. 

But since the amount of work is very very small and we're more concerned about the GC work inflicted by not doing it this way, fine.

Reply:
Yeah, lazy initialization and a more organized interning strategy is definitely on my radar! But anything set post-init will need synchronization, and at least the naive approaches (the usual mutex, etc) have proven to be bottlenecks when invoked in hot spots... so the jury is still out on the best approach there.